CHANGELOG.asciidoc

```diff
@@ -31,6 +31,7 @@ image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/ima
 * Changed Gremlin grammar to make use of `g` to spawn child traversals a syntax error.
 * Added `unexpected-response` handler to `ws` for `gremlin-javascript`
 * Fixed bug in `TinkerTransactionGraph` where a read-only transaction may leave elements trapped in a "zombie transaction".
+* Changed console to consider typos and empty lines in plugins.txt by skipping over or clarifying as invalid.
```

Comment:
```suggestion
* Fixed issue in `gremlin-console` where it couldn't accept plugin files that included empty lines or invalid plugin names.
```
Explain what was fixed, not how it was fixed since this is a user facing file.